test/index.test.ts

```diff
@@ -15,12 +15,11 @@
  * limitations under the License.
  */
 
-import { promises as fs } from 'fs';
+import plugin = require('../src');
```

Comment:
Why are we switching this from `import` to `require`?

Reply:
Per https://www.typescriptlang.org/docs/handbook/modules.html#export--and-import--require:

> When exporting a module using `export =`, TypeScript-specific `import module = require("module")` must be used to import the module.

Reply:
Wow, that's very strange.